view-creator-framework/build.gradle.kts

```diff
@@ -71,6 +73,8 @@ dependencies {
   annotationProcessor("org.projectlombok:lombok:1.18.26")
   implementation("org.slf4j:slf4j-api:2.0.7")
   implementation("com.typesafe:config:1.4.2")
+  implementation("io.airlift:aircompressor:0.27")
```

Comment:
This would be a constraint rather than a direct dependency. We want to say "I won't accept a version of aircompressor below `0.27`" not "I need aircompressor at 0.27". A constraint does nothing if the dependency isn't already on the classpath.